node/src/main/scala/com/wavesplatform/state/Blockchain.scala

```diff
@@ -93,6 +93,11 @@ trait Blockchain {
   def collectLposPortfolios[A](pf: PartialFunction[(Address, Portfolio), A]): Map[Address, A]
 
   def invokeScriptResult(txId: TransactionId): Either[ValidationError, InvokeScriptResult]
+
+  /**
+    * Retrieves Blake2b256 signature or VRF (after feature activation) for the block at height.
```

Comment:
That's not quite true: it's NXT-style generation signature (which is a _hash_ of miner's public key and previous generation signature)